tests/functional/pv/space_reclaim/test_staggered_cronjob_scheduling.py

```
@@ -328,3 +350,785 @@ def _assert_stagger_spread(self, timestamps: dict) -> None:
+            matching = [
+                j
+                for j in all_jobs
+                if j["metadata"]["name"].startswith(f"{cj_obj.resource_name}-")
+            ]
+            if len(matching) > jobs_before_count:
+                newest = max(
+                    matching,
+                    key=lambda j: j["metadata"]["creationTimestamp"],
+                )
+                ts_str = newest["metadata"]["creationTimestamp"]
+                job_time = datetime.strptime(ts_str, "%Y-%m-%dT%H:%M:%SZ")
```

Ruff (0.15.15)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)